src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.SslProtocols.cs

```diff
@@ -101,15 +95,10 @@ public static IEnumerable<object[]> GetAsync_AllowedSSLVersion_Succeeds_MemberDa
         }
 
         [OuterLoop] // TODO: Issue #11345
```

Comment:
You removed some of the other OuterLoops... did you explicitly choose to leave this one?